plgconfig.js

```diff
@@ -7,7 +7,7 @@ var config = {
   plgCategory:    'MEI Export',
   pluginFilename: 'sibmei4.plg',
   linkLibraries: [
-    'libmei4.plg', 'sibmei4_batch_mxml.plg', 'sibmei4_batch_sib.plg', 'sibmei4_test_runner.plg', 'sibmei4_extension_test.plg'
+    'libmei4.plg', 'sibmei4_batch_mxml.plg', 'sibmei4_batch_sib.plg', 'sibmei4_test_runner.plg', 'sibmei4_extension_test.plg', 'sibmei4_legacy_extension_api_v1_test.plg'
```

Comment:
I’m really not a huge fan of copying these examples to the installation by default. Is this necessary? It just seems to add a lot of plugins that most users will never use.

Reply:
This is necessary during development because when working on the extensions, we would otherwise have to copy&paste them manually.

But I agree it was a mistake to include the test code in the last release.  That should be avoided.

Reply:
Can you just change it locally and not commit your changes to the config?

Reply:
Everybody running the tests would have to do that.  Otherwise the tests will crash because of missing plugins.

Reply:
I think that’s something that can be documented, no? The only people running the tests are devs, and  it would clean up the UI for everyone else.

Reply:
But everybody else should probably use a release version where we only package the necessary files (which we didn't do last time).  If we can avoid working with a permanently "dirty" working directory, I'd highly welcome that.  If we stash something, checkout another branch, continue working, then we end up with the wrong version of the test plugins.  We have to remember to change the file every time we switch a branch.  Or changed files will accidentally not be updated, we test with the wrong version of the files and don't notice they are broken.

For our inhouse plugins, our own compiler creates test and release versions of the plugins.  That could be possible for Sibmei as well, but I don't think that's worth the effort.